docs/MAINTAINERS.md

```diff
@@ -31,6 +31,7 @@ Below is the list of Snap Maintainers, they are largely responsible for approvin
 | Matt Broberg      | @mbbroberg       |
 | Patryk Matyjasek  | @PatrykMatyjasek |
 | Rashmi Gottipati  | @rashmigottipati |
+| Klaudiusz Dembler | @kdembler        |
```

Comment:
please place Klaudiusz in alphabetical order :)

Reply:
done